confluent_kafka/avro/__init__.py

```diff
@@ -1,6 +1,8 @@
 import logging
 import sys
 
+from confluent_kafka.cimpl import Producer
```

Comment:
Possibly related to this:
https://travis-ci.org/confluentinc/confluent-kafka-python/jobs/175863152#L584